lib/css_splitter/sprockets_engine.rb

```diff
@@ -10,6 +10,15 @@ def self.engine_initialized?
     def prepare
     end
 
+    def self.call(input)
+      filename = input[:filename]
+      data     = input[:data]
+      context  = input[:environment].context_class.new(input)
+
+      data = self.new(filename) { data }.render(context, {})
```

Comment:
i find this syntax a little hard to understand.  could this be expressed in a more simple way?